pkg/exchange/binance/stream.go

```diff
@@ -544,8 +806,18 @@ func (s *Stream) read(ctx context.Context) {
 			
 			case *MarkPriceUpdateEvent:
 				s.EmitMarkPriceUpdateEvent(e)
-                        case *ContinuousKLineEvent:
-                                 s.EmitContinuousKLineEvent(e)
+
+            case *ContinuousKLineEvent:
+                s.EmitContinuousKLineEvent(e)
+
+			case *AccountUpdateEvent:
+				s.EmitAccountUpdateEvent(e)
+
+			case *AccountConfigUpdateEvent:
+				s.EmitAccountConfigUpdateEvent(e)
+
+			case *OrderTradeUpdateEvent:
+				s.EmitOrderTradeUpdateEvent(e)
```

Comment:
can you reformat this part? you can use gofmt or GoLand to help you reformat.

Reply:
Of course, reformatted.